Kickstarter-iOS/Views/RewardCardContainerView.swift

```diff
@@ -48,10 +48,8 @@ public final class RewardCardContainerView: UIView {
       |> \.layoutMargins .~ .init(all: Styles.grid(3))
 
     _ = self.pledgeButton
-      |> checkoutGreenButtonStyle
-
-    _ = self.pledgeButton.titleLabel
-      ?|> checkoutGreenButtonTitleLabelStyle
+      |> roundedGreenButtonStyle
+      |> UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode .~ .byTruncatingTail
```

Comment:
Here, to avoid the use of lens and to keep consistent with the styling code on `LoginTouViewController`, we can use:
```
_ = self.pledgeButton.titleLabel
      ?|> \.lineBreakMode .~ .byTruncatingTail
```